conda_build/api.py

```diff
@@ -205,8 +205,8 @@ def build(recipe_paths_or_metadata, post=None, need_source_download=True,
 
     if not absolute_recipes:
         raise ValueError('No valid recipes found for input: {}'.format(recipe_paths_or_metadata))
-    return build_tree(absolute_recipes, config, stats, build_only=build_only, post=post,
-                      notest=notest, need_source_download=need_source_download, variants=variants)
```

Comment:
Was the removal of this parameter intentional?  It is used in the build_tree function.

Reply:
It was unused so I removed it.

Reply:
https://github.com/conda/conda-build/pull/3932/files#diff-ac55d744935b1f37a3369258b88e8f1fL2771-L2772

Reply:
Nevermind, read further and now see that this parameter is not used.